browser-test/test.oauth2.ts

```diff
@@ -111,25 +110,22 @@ describe('Browser OAuth2 tests', () => {
     });
   });
 
-  it('should generate a valid consent page url', done => {
+  it('should generate a valid consent page url', () => {
     const opts = {
       access_type: ACCESS_TYPE,
       scope: SCOPE,
       response_type: 'code token'
     };
 
     const generated = client.generateAuthUrl(opts);
-    const parsed = url.parse(generated);
-    if (typeof parsed.query !== 'string') {
-      throw new Error('Unable to parse querystring');
-    }
-    const query = qs.parse(parsed.query);
+    // can't use URL class in webpack, so parsing URL manually
+    const queryString = generated.replace(/^.*?\?(.*?)(?:#.*)?$/, "$1");
```

Comment:
wait wat - WHATWG Url is the new hot in node because it's a browser class.
https://developer.mozilla.org/en-US/docs/Web/API/URL